requirements.txt

```diff
@@ -4,3 +4,7 @@ coveralls
 pyRestTable
 pandas
 xlrd
+ophyd
+bluesky
+historydict
+databroker[all]
```

Comment:
We normally separate the dev requirements (`versioneer`, `coverage`, `coveralls`, `sphinx`, etc.) and the run requirements (such as those added in this PR + 3 above). It's mostly for convenience, up to you.

Also, we try to keep this list sorted - also, for convenience and easier searching of already added libraries.

Reply:
Yes, just alphabetical order is fine.

In the latest [suitcase-* packages](https://github.com/NSLS-II/suitcase-cookiecutter/tree/master/suitcase-%7B%7B%20cookiecutter.subproject_name%20%7D%7D) and [scientific python cookiecutter](https://github.com/NSLS-II/scientific-python-cookiecutter/tree/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D) we tend to call it `requirements.txt` and `requirements-dev.txt` (while we still use `dev-requirements.txt` in bluesky, will be changed later). It's again for usability, so that when typing `pip install -r requirements<TAB>` it will show all possible files in console.

Reply:
+1 for syntax Yoda